### DISCUSSION TITLE
Can any body explain this why?  cell twinkling when scroll using virtualize config

### DISCUSSION THREAD

**Discussion by putrasatr** [rsuite/rsuite-table#385] [Q&A]
https://user-images.githubusercontent.com/76176170/202641893-c4a11c6a-e327-4870-afa3-27a22f50668d.mov

This is my code :
<img width="588" alt="Screen Shot 2022-11-18 at 11 41 27" src="https://user-images.githubusercontent.com/76176170/202641930-8ba7eb5d-7725-438c-a2fd-4482e92918b4.png">

**Comment by simonguo** [comment]
Hi, what react and rsuite version are you using?

**Reply by putrasatr** [reply]
i use 
"react": "18.2.0",
"rsuite-table": "^5.7.2"

But I just found the cause, apparently it's due to using @heroiconreact as table value. in my case i wrapped the icon component in a cursor-pointers-none div and everything goes back to normal.

**Reply by simonguo** [reply]
If you use React 18, it is recommended to upgrade rsuite-table to the latest version [5.8.1](https://github.com/rsuite/rsuite-table/releases/tag/5.8.1)
